lib/src/cli/atlas_apps/deployapps_command.dart

```diff
@@ -67,9 +67,13 @@ class DeployAppsCommand extends Command<void> {
     final apps = await client.getOrCreateApps();
 
     print('App import is complete. There are: ${apps.length} apps on the server:');
+    List<String> listApps = [];
     apps.forEach((_, value) {
       print("  App '${value.name}': '${value.clientAppId}'");
+      listApps.add(value.appId);
     });
+    print("appIds: ");
+    stdout.write(listApps.join(","));
```

Comment:
```suggestion
   
```

Reply:
I printed these Ids on a new line by reason. It is easier to read them as an output of the command. I removed already the usage of this output, but I think they could help us for something else.

Reply:
ok thanks